src/main/resources/migrations/article/init-table.xml

```diff
@@ -11,7 +11,7 @@
             <column name="id" type="bigint" autoIncrement="true">
                 <constraints primaryKey="true" nullable="false" unique="true"/>
             </column>
-            <column name="tags" type="varchar"/>
+            <column name="tag" type="varchar"/>
```

Comment:
stop changing existing migrations!
bad practice

Reply:
will be done:)